keyboards/ergodox/keymaps/narze/config.h

```diff
@@ -0,0 +1,21 @@
+#ifndef CONFIG_H_
+#define CONFIG_H_
+
+#include "../../config.h"
```

Comment:
This line is not actually needed, and should be removed. 

```suggestion
```

Specifically, the build script automatically includes all of the config.h files, so this is not necessary.